server/lib/nl/common/utils.py

```diff
@@ -60,11 +60,14 @@ def event_existence_for_place(place: str, event: types.EventType,
     counters.timeit('event_existence_for_place', start)
     cur_year = datetime.datetime.now().year
     prev_year = str(cur_year - 1)
+    two_years = str(cur_year - 2)
```

Comment:
Thanks for this.

Should we match the Earth behavior?  Earth also might have a ton of data.

Reply:
Sorry, what do you mean? This change just adds two more years to the list of allowed years in the check & doesn't add more data processing?